sdks/python/src/opik/message_processing/batching/base_batcher.py

```diff
@@ -0,0 +1,49 @@
+import threading
+import time
+import abc
+
+from typing import List, Callable
+from .. import messages
+
+
+class BaseBatcher(abc.ABC):
+    def __init__(
+        self,
+        flush_callback: Callable[[messages.BaseMessage], None],
+        max_batch_size: int,
+        flush_interval: float,
```

Comment:
You need to make clear the time unit of the flush internal. After that, probably makes sense to use an integer type and go with milliseconds.